### PR TITLE
QRadar Generic playbook and updated incident type.

### PR DESCRIPTION
**Pull request by David-BMS**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/32847

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Michal-Vardi** [comment]
@David-BMS 
Done

File: Packs/QRadar/Playbooks/playbook-QRadar_Generic.yml [hunk 1]
```diff
@@ -0,0 +1,1890 @@
...
+  value:
+    simple: 'Adjustment/Exclusion for offense '
+  required: false
+  description: The subject of the email to send to the SIEM admin.
+  playbookInputQuery:
+- key: UseCustomSeveritySettings
+  value:
+    simple: "false"
+  required: false
+  description: Determines whether to use the default mapping as provided in the QRadar
```
Comment (David-BMS) [inline Packs/QRadar/Playbooks/playbook-QRadar_Generic.yml:1859]: @Michal-Vardi Please review the new input

File: Packs/QRadar/ReleaseNotes/1_3_0.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+#### Incident Types
+- **Qradar Generic**
+Assigned the **QRadar Generic** playbook as the default playbook for this incident type.
+
+#### Playbooks
+##### QRadar Generic
+The default playbook for working with the QRadar V2 integration for basic SIEM/SOC operations.
+#### Classifiers
+##### QRadar - Generic Incoming Mapper
+- Added mapping for the severity field bases on the magnitude value. The XSOAR incident severity field values range is 0-4 where<br/>0 - Informational<br/>1 - Low<br/>2 - Medium<br/>3 - High<br/>4 - Critical<br/><br/>A scale is required to translate the value of the magnitude field value to a valid incident severity value. The default scale is 1,1,1,2,2,2,2,3,3,3 <br/>The meaning of the default scale is that values 1-3 of magnitude will be translated to low severity \(positions 1-3 in the scale\), values 4-7 will be translated to medium severity \(positions 4-7 in the scale\) and values 8-10 will be translated to high severity \(positions 8-10 in the scale\).
```
Comment (David-BMS) [inline Packs/QRadar/ReleaseNotes/1_3_0.md:11]: @Michal-Vardi And this release note which refers to the change in the mapper and not the playbook

**Comment by David-BMS** [comment]
@michalgold Adjustments we made as requested, please approve

**Comment by David-BMS** [comment]
@yaron-libman FYI

**Review by michalgold — APPROVED** [review]

**Pull request merged**